### PR TITLE
MicrosoftStudios.MinecraftEducationEdition version 1.14.70.0

### PR DESCRIPTION
**Pull request by One11Onee**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/27401)

### CODE REVIEW

File: manifests/m/MicrosoftStudios/MinecraftEducationEdition/1.14.70.0/MicrosoftStudios.MinecraftEducationEdition.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+PackageIdentifier: MicrosoftStudios.MinecraftEducationEdition
+PackageVersion: 1.14.70.0
+InstallerSwitches:
+  Silent: error
+  SilentWithProgress: error
+Installers:
+- Architecture: x86
+  InstallerType: exe
+  InstallerUrl: https://aka.ms/downloadmee-desktopApp
+  InstallerSha256: 04B3B00658399D717958CD0BDF675B2FF2CF53C165694A8E6C52B052850C2013
```
Comment (jedieaston) [inline manifests/m/MicrosoftStudios/MinecraftEducationEdition/1.14.70.0/MicrosoftStudios.MinecraftEducationEdition.installer.yaml:13]:
```suggestion
  InstallerSha256: 04B3B00658399D717958CD0BDF675B2FF2CF53C165694A8E6C52B052850C2013
  InstallerSwitches:
      Silent: "/exenoui AI_DESKTOP_SH=1 /qn"
      SilentWithProgress: "/exenoui AI_DESKTOP_SH=1 /qb"
```

File: manifests/m/MicrosoftStudios/MinecraftEducationEdition/1.14.70.0/MicrosoftStudios.MinecraftEducationEdition.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: MicrosoftStudios.MinecraftEducationEdition
```
Comment (jedieaston) [inline manifests/m/MicrosoftStudios/MinecraftEducationEdition/1.14.70.0/MicrosoftStudios.MinecraftEducationEdition.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: Mojang.MinecraftEducationEdition
```

File: manifests/m/MicrosoftStudios/MinecraftEducationEdition/1.14.70.0/MicrosoftStudios.MinecraftEducationEdition.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: MicrosoftStudios.MinecraftEducationEdition
```
Comment (jedieaston) [inline manifests/m/MicrosoftStudios/MinecraftEducationEdition/1.14.70.0/MicrosoftStudios.MinecraftEducationEdition.installer.yaml:4]:
```suggestion
PackageIdentifier: Mojang.MinecraftEducationEdition
```

(Note: You'll have to move it to the correct folder as well).

File: manifests/m/MicrosoftStudios/MinecraftEducationEdition/1.14.70.0/MicrosoftStudios.MinecraftEducationEdition.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: MicrosoftStudios.MinecraftEducationEdition
```
Comment (jedieaston) [inline manifests/m/MicrosoftStudios/MinecraftEducationEdition/1.14.70.0/MicrosoftStudios.MinecraftEducationEdition.yaml:4]:
```suggestion
PackageIdentifier: Mojang.MinecraftEducationEdition
```

**Review by jedieaston — CHANGES_REQUESTED** [review]
Thanks for the addition! Just a few things to fix.

File: manifests/m/MicrosoftStudios/MinecraftEducationEdition/1.14.70.0/MicrosoftStudios.MinecraftEducationEdition.installer.yaml (hunk 1 above)
Comment (QingSaYa) [reply manifests/m/MicrosoftStudios/MinecraftEducationEdition/1.14.70.0/MicrosoftStudios.MinecraftEducationEdition.installer.yaml:13]: that is same, i can't change that

**Review by One11Onee — COMMENTED** [review]
the InstallerSha256 is same, I can't change that

File: manifests/m/MicrosoftStudios/MinecraftEducationEdition/1.14.70.0/MicrosoftStudios.MinecraftEducationEdition.installer.yaml (hunk 1 above)
Comment (jedieaston) [reply manifests/m/MicrosoftStudios/MinecraftEducationEdition/1.14.70.0/MicrosoftStudios.MinecraftEducationEdition.installer.yaml:13]: I added the silent install switches?

**Pull request closed without merging**